zerver/tests/test_auth_backends.py

```diff
@@ -1217,6 +1217,9 @@ def test_social_auth_registration_without_is_signup(self) -> None:
 
     def test_social_auth_registration_without_is_signup_closed_realm(self) -> None:
         """If the user doesn't exist yet in closed realm, give an error"""
+        realm = get_realm("zulip")
+        realm.emails_restricted_to_domains = True
+        realm.save()
```

Comment:
I generally prefer using the `actions.py` functions for this purpose, since that'll be more correct if we need to do something more complicated when changing the setting.  

I've sorta been meaning to do an audit of our tests to clean out this sort of thing, but figured I'd highlight it since you write a lot of new tests :)

Reply:
@timabbott Fixed, `do_set_realm_property` is the appropriate function I think? I'll try to look through other tests too.

Reply:
yes, that's correct.  

Probably worth coordinating with @showell since he submitted a 600-line test cleanup PR and it'd be a shame to merge conflict two big testing migrations, but I think basically `git grep realm.save zerver/tests` should be cleared out and linted against.